BO4E/BO/SteuerbareRessource.cs

```diff
@@ -50,5 +52,15 @@ public class SteuerbareRessource : BusinessObject
         [JsonPropertyName("zugeordnetMSBCodeNr")]
         [ProtoMember(12)]
         public string? ZugeordnetMSBCodeNr { get; set; }
+
+        /// <summary>
+        /// Produkt-Daten der Steuerbaren Ressource
+        /// </summary>        
+        [JsonProperty(Required = Required.Default, Order = 13, PropertyName = "konfigurationsprodukte")]
+        [JsonPropertyName("konfigurationsprodukte")]
+        [ProtoMember(13)]
+        [JsonPropertyOrder(13)]
```

Comment:
ist die Order hier nicht redundant in den `JsonProperty` ?

Reply:
gute frage! JsonPropertyOrder auch kann weg. 
ganze project ist so ;) 
[mussen wir ein ](https://github.com/Hochfrequenz/BO4E-dotnet/issues/402) hab ein Issue dafür aufgemacht.